lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -371,14 +374,46 @@ public void indexObjects(
                 pw.println(UObject.transformObjectToString(doc));
             }
             pw.close();
-            makeRequestBulk("POST", indexName, tempFile);
+            Response response = makeRequestBulk("POST", indexName, tempFile);
+
+            verify(response);
+
             updateLastVersionsInData(indexName, pguid, lastVersion);
         } finally {
             tempFile.delete();
         }
         refreshIndex(indexName);
     }
-    
+
+
+    private void verify(Response response) throws IOException {
+        // convert JSON string to Map
+        String responseJsonStr = EntityUtils.toString(response.getEntity());
+
+        ObjectMapper mapper = new ObjectMapper();
+        final Map<String, Object> responseMap = mapper.readValue(responseJsonStr,
+                new TypeReference<Map<String, Object>>() {});
+
+        // check for and collect response data from index operations that failed
+        List errorItems = new ArrayList();
```

Comment:
Should there be a max size for this list? The exception text could be really big for a large load. Maybe log everything but only throw a portion of the list?

Reply:
It won't be larger than the load itself so I am not sure if I should just put some arbitrary number. Logging would be great but at the moment there is no logging in ElasticIndexingStorage. I will create a separate PR for that.

Reply:
The load size could be up to 100k- that's a really really large exception string. 

Logging should be easy - just use slf4j and logback classic (which may already be in the ant file) and do 

```
	private void logInfo(final String format, final Object... params) {
		LoggerFactory.getLogger(getClass()).info(format, params);
	}
	
	private void logErr(final String format, final Object... params) {
		LoggerFactory.getLogger(getClass()).error(format, params);
	}
```

Reply:
That'll be automatically be intercepted by the built in logger in the SearchAPI and will just go to stdout for the workers, I believe.

Reply:
I have removed the exception and added a statement to log the errors.